app/scripts/services/quota.js

```diff
@@ -277,6 +279,45 @@ angular.module("openshiftConsole")
         return _.some(quotas, isExceeded) || _.some(clusterQuotas, isExceeded);
     };
 
+    // Same as above but only looking at storage items: requests.storage, persistentvolumeclaims
+    //   Warn if you are at quota or over any storage quota for any resource.
+    var isAnyStorageQuotaExceeded = function(quotas, clusterQuotas, requestedStorage) {
```

Comment:
instead of copying the function above I would rather have an optional third parameter to isAnyQuotaExceeded that is a list of quota keys to check, then your if check for whether you check that particular quotaKey is something like 

```
if (!typesToCheck || _.includes(typesToCheck, quotaKey) {
  hard = usageValue(hard);
  // etc...
```

then if you wanted to you could add a function for "isAnyStorageQuotaExceeded" that looks more like:
```
var isAnyStorageQuotaExceeded = function(quotas, clusterQuotas) {
  return isAnyQuotaExceeded(quotas, clusterQuotas, ['requests.storage', 'persistentvolumeclaims]');
};
```

The only case where you were actually using requestedStorage was for the field validity check on requests.storage.  For that purpose I would rather see a generic function written that checks a particular quotaKey's `hard <= (used + request)`